main/styles/local-osm-liberty-custom.json

```diff
@@ -0,0 +1,3787 @@
+{
+  "version": 8,
+  "name": "OSM Liberty",
+  "metadata": {
+    "mapbox:type": "template"
+  },
+  "sources": {
+    "openmaptiles": {
+      "type": "vector",
+      "url": "http://localhost:8080/data/v3.json"
+    }
+  },
+  "sprite": "https://rawgit.com/lukasmartinelli/osm-liberty/gh-pages/sprites/osm-liberty",
+  "glyphs": "https://orangemug.github.io/font-glyphs/glyphs/{fontstack}/{range}.pbf",
```

Comment:
To keep in mind for later: Eventually it would be good to get rid of such dependencies, because I don't know how certain we can be that these sources will always be available in the future. I think it would be possible both technically and legally to create a local copy.